### DISCUSSION TITLE
Running wallaby features in VS Code

### DISCUSSION THREAD

**Discussion by totaltrash** [keathley/wallaby#717] [Q&A]
Anyone using vs code and know of a way to run wallaby features from the code editor?

Not 100% sure, I might have inadvertently installed a testing extension (I recently went wild with extensions for docker and remote development, though there are no installed testing extensions listed for me), and I can now run ex unit tests by hitting the green arrow, but it's not offering the same for features:

![image](https://user-images.githubusercontent.com/637350/206048059-508eaeab-2c97-4f01-b7b9-d75463d4725f.png)

**Comment by buildreactive** [comment]
Just bumping this. Same problem here. I'm using the `ElixirLS: Elixir support and debugger` extension, which provides support for "Test discovery, running and debugging via Test Explorer." Unfortunately this does not appear to work with Wallaby; `feature` tests don't show up – likely the hinting is looking for "test" not "feature" so it just completely missed them.
